src/app/series/directives/series-podcast.component.html

```diff
@@ -0,0 +1,65 @@
+<form [ngSwitch]="state">
+
+  <publish-fancy-field label="Series Podcast" *ngSwitchCase="'new'">
+    <div class="fancy-hint">You must save your series before you can create
+      a podcast distribution for it.</div>
+  </publish-fancy-field>
+
+  <publish-fancy-field label="Series Podcast" *ngSwitchCase="'creating'">
+    <div class="fancy-hint">No podcast has been configured for this series.</div>
+    <button (click)="createDistribution()">Create Podcast</button>
+  </publish-fancy-field>
+
+  <div *ngSwitchCase="'editing'">
+
+    <publish-fancy-field label="PRX Feeds" *ngIf="podcast?.id">
+      <div class="fancy-hint">There should really be some text here describing
+        exactly what these are and what you might do with them.</div>
+      <div class="feed-urls">
+        <publish-fancy-field label="Published Feed" small=1>
+          <input type="text" readonly [ngModel]="podcast?.publishedUrl" name="publishedUrl" #pubFeed/>
+          <button [publishCopyInput]="pubFeed">Copy</button>
+          <a class="button" target="_blank" [href]="podcast?.publishedUrl">Open Link</a>
+        </publish-fancy-field>
+        <publish-fancy-field label="Preview Feed" small=1>
+          <input type="text" readonly [ngModel]="podcast?.previewUrl" name="previewUrl"/>
+          <a class="button" target="_blank" [href]="podcast?.previewUrl">Open Link</a>
+        </publish-fancy-field>
+      </div>
+    </publish-fancy-field>
+
+    <publish-fancy-field label="iTunes Categories" required=1>
+      <div class="fancy-hint">Some description of what these are here and
+        maybe also a link to the itunes docs.</div>
+      <div class="span-fields">
+        <publish-fancy-field label="Category" [model]="podcast" name="category"
+          [select]="categories" (change)="setSubCategories()" small=1>
+        </publish-fancy-field>
+        <publish-fancy-field *ngIf="!subCategories.length" label="Sub-Category"
+          [select]="[]" small=1>
+        </publish-fancy-field>
+        <publish-fancy-field *ngIf="subCategories.length" label="Sub-Category"
+          [model]="podcast" name="subCategory" [select]="subCategories" small=1>
+        </publish-fancy-field>
+      </div>
+    </publish-fancy-field>
+
+    <publish-fancy-field label="Explicit" required=1 [model]="podcast" name="explicit" [select]="explicitOpts">
+      <div class="fancy-hint">Do any of your podcast episodes contain explicit material?</div>
+    </publish-fancy-field>
+
```

Comment:
Yeah, we should open another ticket for that.  It's a bit complicated since the value of this field is currently stored in feeder, not CMS.  But worth looking into.